streams/src/main/java/org/apache/kafka/streams/processor/TopologyBuilder.java

```diff
@@ -21,6 +21,10 @@
 import org.apache.kafka.common.serialization.Serializer;
 import org.apache.kafka.common.utils.Utils;
 import org.apache.kafka.streams.errors.TopologyBuilderException;
+import org.apache.kafka.streams.kstream.internals.CacheEnabledProcessor;
```

Comment:
Originally we hierarchize the packages to only let `kstreams` sub-package to depend on `processor`, but not vice versa, since mutual dependency will make refactoring in the future to be difficult. If this this DSL-only feature HAS to be leaked into the topology builder, then how about just moving them to to `processor`, since we may expose them as general functionalities in the processor APIs in the future?